test/PhpEnvironment/ResponseTest.php

```diff
@@ -100,4 +101,37 @@ public function testCanExplicitlySetVersion()
         $this->expectException(InvalidArgumentException::class);
         $response->setVersion('laminas/2.0');
     }
+
+    /**
+     * @runInSeparateProcess
+     */
+    public function testSendHeadersHeadersNotAlreadySent()
+    {
+        $response = new Response();
+        $this->assertInstanceOf(Response::class, $response->sendHeaders());
+    }
+
+    public function testSendHeadersHeadersAlreadySentPassInvalidHandler()
+    {
+        $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage(
+            'Handler must be callable with passed response object in invokable parameter, received string'
+        );
```

Comment:
The various `expectException*()` methods should be called immediately before the method call that is expected to raise them; otherwise, we may get a false positive if other statements raise the same errors. (This is true with your other tests as well.)

Reply:
implemented.